unlock-protocol.com/src/pages/_document.js

```diff
@@ -21,6 +44,8 @@ export default class MyDocument extends Document {
           <Fonts />
           <link rel="shortcut icon" href="/static/favicon.ico" />
           {this.props.styleTags}
+          <script dangerouslySetInnerHTML={this.props.unlockTag} />
```

Comment:
you can use `<script>` tags directly inside `<Head>`

Reply:
I tried but it does not seem to work :/

Reply:
```> Using external babel configuration
> Location: "/Users/julien/repos/unlock/unlock-protocol.com/.babelrc"
[ error ] ./pages/_document.js
SyntaxError: /Users/julien/repos/unlock/unlock-protocol.com/src/pages/_document.js: Unexpected token (48:8)

  46 |           {this.props.styleTags}
  47 |           <script>(function(d, s) {
> 48 |         var js = d.createElement(s),
     |         ^
  49 |         sc = d.getElementsByTagName(s)[0];
  50 |         js.src="https://paywall.unlock-protocol.com/static/unlock.1.0.min.js";
  51 |         sc.parentNode.insertBefore(js, sc); }(document, "script"))</script>
events.js:183
      throw er; // Unhandled 'error' event
      ^
```

Reply:
you are right!

Reply:
it also may work (in a follow-up PR, don't want to block this any further) to copy the approach taken here: https://github.com/unlock-protocol/unlock/blob/master/paywall/src/components/content/NewDemoContent.tsx#L27
and here: https://github.com/unlock-protocol/unlock/blob/master/paywall/src/components/content/NewDemoContent.tsx#L55